### ISSUE TITLE
Misplaced parenthesis on page 206 (line 73 in source file 11_language.txt)? 

### ISSUE THREAD

**Issue by xgfd** [marijnh/Eloquent-JavaScript#159]
Should the following code
```
do(define(x, 10),
   if(>(x, 5)), //an extra ')' here?
   print("large"),
   print("small")) //should go here?
```
be
```
do(define(x, 10),
   if(>(x, 5),
      print("large"),
      print("small")))
```

**Comment by marijnh** [comment]
You are right. Thanks for spotting that. Fixed in patch 6a5b6a6

**Comment by xgfd** [comment]
NP. Thanks for the prompt fix.
